dmoj/executors/filesystem_policies.py

```diff
@@ -0,0 +1,114 @@
+import os
+from typing import List, Union
+
+
+class ACCESS_MODE:
```

Comment:
Nit: should be `AccessMode` for consistency. Also, this should probably inherit from `Enum`, since it is one.